docs/topics/feed-exports.rst

```diff
@@ -347,6 +370,19 @@ as a fallback value if that key is not provided for a specific feed definition:
 
 -   ``fields``: falls back to :setting:`FEED_EXPORT_FIELDS`.
 
+-   ``item_classes``: list of acceptable :ref:`item classes <topics-items>` that should be exported to the feed storage.
+
+    If no items declared then there will be no filtering based on item_classes. There is no fallback value.
+
+    .. versionadded:: 2.X.X
+
+-   ``item_filter``: a :ref:`filter class <item-filter>` based on ItemChecker class to filter items before they are
+    exported to the feed storage.
```

Comment:
```suggestion
-   ``item_filter``: a :ref:`filter class <item-filter>` to filter items before they are
    exported to the feed storage.
```

The classes do not need to subclass `ItemChecker`, which was a possible interpretation here. This way we keep it shorter, and users can read the linked section for details on how to implement such a class.